__tests__/workers/notifications.ts

```diff
@@ -158,7 +158,7 @@ it('should NOT add devcard unlocked notification if user had already reached the
       reputation: 40,
     },
   });
-  expect(actual).toEqual(null);
+  expect(actual).toEqual(undefined);
```

Comment:
There was no point in returning `null` so now it's just `undefined`, worker expects falsy either way with `!value` check.

Reply:
I personally also use `toBeNull` for `null`, and `toBeUndefined` for `undefined`. Bit more explicit

Reply:
Yeah, true